test/e2e/view-capacity-region.js

```diff
@@ -21,7 +21,7 @@ describe('View your caseload capacity flow', () => {
     await regionCasloadLink.click()
     pageSubtitle = await $('.govuk-caption-xl')
     pageSubtitle = await pageSubtitle.getText()
-    expect(pageSubtitle).to.equal('Region')
+    expect(pageSubtitle).to.equal('Offender Manager')
```

Comment:
This is the region caseload capacity view, based on what I can understand from the URL and the test filename

https://workload-measurement-preprod.hmpps.service.justice.gov.uk/probation/region/31/caseload-capacity

There isn't anything offender manager specific here